torchci/pages/cost_analysis.tsx

```diff
@@ -576,6 +594,17 @@ export default function Page() {
             <FaInfoCircle />
           </Typography>
         </Tooltip>
+        <Tooltip title="Copy Permalink">
```

Comment:
There's a copy link component already.  Can you use that instead?

See:

https://github.com/pytorch/test-infra/blob/main/torchci/components/CopyLink.tsx

Reply:
updated